apps/src/applab/appStorage.js

```diff
@@ -38,6 +38,10 @@ var handleGetKeyValue = function(onSuccess, onError) {
   if (this.readyState !== 4) {
```

Comment:
Do you know why they need to do  var DONE = this.DONE || 4? Is it because only some browsers set this.DONE?